coral-schema/src/main/java/com/linkedin/coral/schema/avro/SchemaUtilities.java

```diff
@@ -217,32 +217,6 @@ static void appendField(@Nonnull String fieldName, @Nonnull RelDataType fieldRel
     }
   }
 
-  static boolean isFieldNullable(@Nonnull RexCall rexCall, @Nonnull Schema inputSchema) {
-    Preconditions.checkNotNull(rexCall);
-    Preconditions.checkNotNull(inputSchema);
-
-    // the field is non-nullable only if all operands are RexInputRef
```

Comment:
do we know why we were adding this ? 

It seems to me that, this PR invalidate this nullable-check method and forcing the nullability to come from `RelDataType's#isNullable` method. So if there are some scenario previously relying on this method to return `nullable` now could return different result ? Just thinking out loud, let me know if I miss something here

Reply:
This method is by design to infer nullability of UDF return types. The reason not using RelDataType's#isNullable is that it is not initialized with proper nullability information at the beginning due to various limitations.